app/locales/es/index.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Are these strings that survived from a previous translation?  I was thinking it might be most useful to have the translation files full of the full English phrases -- then the translator just needs to walk through the single file and translate as they go.  But I am presuming about how this gets done. @alpita-masurkar ?

Reply:
1. Yep, these are those strings that survived.
2. I too would much prefer the english as the key.

I could keep keys as they are for now in this PR, and do (2) in a follow up if @alpita-masurkar is ok with that.

Reply:
Removed all but ht and en. Will try english keying in a follow up